### ISSUE TITLE
Irregularity in the design 

### ISSUE THREAD

**Issue by nitin10s** [CircuitVerse/CircuitVerseDocs#219]
The "previous" and "next" links have different opacities apparently for no reason. They should have the same opacity level and design.

![image (6)](https://user-images.githubusercontent.com/16988558/78049623-e7d5e780-7398-11ea-8ebf-6053c0c9246c.png)

**Comment by nitin10s** [comment]
might be? How do you know?

**Comment by Ananthavijay** [comment]
Actually it is just assumption. One will go through the docs in the order. So the previous page would be read and hence it is not highlighted

**Comment by nitin10s** [comment]
Nope, this can probably act as a false signifier as well, so should not be used.

**Comment by chehak123** [comment]
I would like to work on it

**Comment by Kartikay304** [comment]
I would like to work on this issue.

**Comment by Yana-do-code** [comment]
Closing as outdated .
